### PR TITLE
Updated Using token exchange chapter #1495

### PR DESCRIPTION
**Pull request by agagancarczyk**
Update for Using token exchange chapter keycloak/keycloak-ui#1495

Notes: 
- mod-auth-mellon-config-download.png was not updated as it doesn't seem to be implemented in the new Admin UI

### CODE REVIEW

**Comment by agagancarczyk** [comment]
hi @andymunro could you please review these updates? Thanks, Agnieszka

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 1]
```diff
@@ -158,14 +163,32 @@ This setup page displays.
 .Target Client Exchange Permission Setup
 image:{project_images}/exchange-target-client-permission-setup.png[Target Client Exchange Permission Setup]
 
+ifeval::[{project_community}==true]
 . Define a policy for this permission by clicking the *Authorization* link
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]: I don't see the Authorization link on this screen.

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 2]
```diff
@@ -158,14 +163,32 @@ This setup page displays.
...
+endif::[]
+ifeval::[{project_product}==true]
+. Define a policy for this permission by clicking the *Client details* link
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]: I don't see the Client details link on this screen.
The original step was 
. Click the *Authorization* link and go to the *Policies* tab to create a client policy.

Can you verify this is how it works for the old console?

File: securing_apps/topics/token-exchange/token-exchange.adoc (hunk 1 above)
Comment (agagancarczyk) [reply securing_apps/topics/token-exchange/token-exchange.adoc]: The old image exchange-target-client-permission-setup.png  with the *Authorization* link is under rhsso-images. I updated the procedure as per our discussion.

File: securing_apps/topics/token-exchange/token-exchange.adoc (hunk 2 above)
Comment (agagancarczyk) [reply securing_apps/topics/token-exchange/token-exchange.adoc]: The new image exchange-target-client-permission-setup.png with 'Client details' link is under keycloak-images. I updated the procedure as per our discussion.

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 3]
```diff
@@ -164,10 +164,12 @@ This setup page displays.
 image:{project_images}/exchange-target-client-permission-setup.png[Target Client Exchange Permission Setup]
 
 ifeval::[{project_community}==true]
-. Define a policy for this permission by clicking the *Authorization* link
+. Click *Authorization* in the breadcrumbs at the top of the screen.
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
. Click *Client details* in the breadcrumbs at the top of the screen.
```

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 4]
```diff
@@ -164,10 +164,12 @@ This setup page displays.
...
+. Define a policy for this permission.
 endif::[]
 ifeval::[{project_product}==true]
-. Define a policy for this permission by clicking the *Client details* link
+. Click *Client details* in the breadcrumbs at the top of the screen.
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
. Click *Authorization* in the breadcrumbs at the top of the screen.
```

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 5]
```diff
@@ -158,14 +163,34 @@ This setup page displays.
...
+. Create a *Client* Policy by clicking *Create policy* button.
+endif::[]
 +
+ifeval::[{project_community}==true]
 .Client Policy Creation
 image:{project_images}/exchange-target-client-policy.png[]
+endif::[]
+ifeval::[{project_product}==true]
+.Select Policy Type
+image:{project_images}/exchange-target-client-policy-type-selection.png[Client Policy Type Selection]
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]: I think this is out of place and may not be needed.

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 6]
```diff
@@ -456,12 +489,27 @@ This setup page displays.
 .Users Impersonation Permission Setup
 image:{project_images}/exchange-users-permission-setup.png[Users Impersonation Permission Setup]
 
-. Click the *Authorization* link
+ifeval::[{project_community}==true]
+. Click *Authorization* in the breadcrumbs at the top of the screen.
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
. Click *Client details* in the breadcrumbs at the top of the screen.
```

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 7]
```diff
@@ -456,12 +489,27 @@ This setup page displays.
...
+. Define a policy for this permission.
+endif::[]
+ifeval::[{project_product}==true]
+. Click *Client details* in the breadcrumbs at the top of the screen.
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
. Click *Authorization* in the breadcrumbs at the top of the screen.
```

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 8]
```diff
@@ -447,7 +474,13 @@ image:{project_images}/exchange-users-permission-unset.png[User Permissions]
 .Identity Provider Permission
 image:{project_images}/exchange-users-permission-set.png[]
 +
+
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc:479]: Can you remove "In the Admin Console" from line 461.

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 9]
```diff
@@ -456,12 +489,27 @@ This setup page displays.
...
 
 . Go to the *Policies* tab and create a client policy.
 +
+ifeval::[{project_community}==true]
+.Client Policy Creation
+image:{project_images}/exchange-users-client-policy.png[Client Policy Creation]
+endif::[]
+ifeval::[{project_product}==true]
+.Select Policy Type
+image:{project_images}/exchange-users-client-policy-selection.png[Client Policy Type Selection]
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]: I think this is out of place and may not be needed.

**Review by andymunro — COMMENTED** [review]
Thanks, @agagancarczyk.  A few more changes. 
Also, this section needs to be updated for the new console: Granting permission for the exchange

**Comment by agagancarczyk** [comment]
hi @andymunro I have made the requested updates and updated section 7.3.1. It turned out that the issue had already been fixed. Thanks

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 10]
```diff
@@ -445,9 +474,15 @@ image:{project_images}/exchange-users-permission-unset.png[User Permissions]
 . Toggle *Permissions Enabled* to true.
```
Comment (andymunro) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
. Toggle *Permissions Enabled* to *On*.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @agagancarczyk. Looks good. Just one very minor change and then we are done.

**Comment by agagancarczyk** [comment]
Thanks @andymunro , this is done now.

**Review by andymunro — APPROVED** [review]
Okay, great. Thanks for the fast response, @agagancarczyk.

Ready to merge, @ssilvert

**Review by ssilvert — APPROVED** [review]

**Pull request merged**